aws-geo-location/src/main/java/com/amplifyframework/geo/location/service/AmazonLocationService.kt

```diff
@@ -15,91 +15,88 @@
 
 package com.amplifyframework.geo.location.service
 
-import com.amazonaws.ClientConfiguration
-import com.amazonaws.auth.AWSCredentialsProvider
-import com.amazonaws.regions.Region
-import com.amazonaws.services.geo.AmazonLocationClient
-import com.amazonaws.services.geo.model.GetMapStyleDescriptorRequest
-import com.amazonaws.services.geo.model.SearchPlaceIndexForPositionRequest
-import com.amazonaws.services.geo.model.SearchPlaceIndexForTextRequest
+import aws.sdk.kotlin.runtime.auth.credentials.CredentialsProvider
+import aws.sdk.kotlin.services.location.LocationClient
+import aws.sdk.kotlin.services.location.model.GetMapStyleDescriptorRequest
+import aws.sdk.kotlin.services.location.model.SearchPlaceIndexForPositionRequest
+import aws.sdk.kotlin.services.location.model.SearchPlaceIndexForTextRequest
+import aws.smithy.kotlin.runtime.ServiceException
 import com.amplifyframework.geo.location.models.AmazonLocationPlace
 import com.amplifyframework.geo.models.Coordinates
 import com.amplifyframework.geo.models.CountryCode
 import com.amplifyframework.geo.models.Place
 import com.amplifyframework.geo.models.SearchArea
-import com.amplifyframework.util.UserAgent
-import java.nio.ByteBuffer
 
 /**
  * Implements the backend provider for the location plugin using
- * AWS Mobile SDK's [AmazonLocationClient].
+ * AWS Mobile SDK's [LocationClient].
```

Comment:
Should it say Kotlin SDK now?

Reply:
I thought about it, but I was not sure if we needed to be explicit about it and since `LocationClient` is referred from Kotlin package anyways.
Unless you have strong a feeling about it :)